Documentation/index.html

```diff
@@ -44,15 +44,63 @@
+                <li>Reduces app size and dependency on unused parts of the underlying OS (e.g., permissions strings required in Info.plist files) by separating specific active tasks or logical groups of tasks into their own modules, in general (but not necessarily) built on these two frameworks;</li>
+                <li>Is designed with parallel development for other (mobile and web) platforms in mind so that the broadest spectrum of study participants can be reached with the minimum coding effort.</li>
+            </ul>
+            
+            <p>Like ResearchKit, this project is independent of any particular back-end service used to collect study data, record consent, or perform other centralized study functions. Sage Bionetworks is also the lead developer of <a href="http://developer.sagebridge.org">Bridge Server</a>, an open source set of services and SDKs for supporting mHealth research.</p>
+            
+            <p>This project represents the results to date and the ongoing implementation of those goals.</p>
+            <h2 id='documentation' class='heading'>Documentation</h2>
+            
+            <p>The core step/task navigation and results-gathering framework is currently called <a href="Research/index.html">Research</a>.</p>
+            
+            <p>The UI/UX framework is called <a href="ResearchUI/index.html">ResearchUI</a>.</p>
+            
+            <h2 id='license' class='heading'>License</h2>
+            
+            <p>Sage Research SDK is available under the BSD license:</p>
+            
+            <p>Copyright &copy; 2017-2019, Sage Bionetworks
+            All rights reserved.</p>
+            
+            <p>Redistribution and use in source and binary forms, with or without
+            modification, are permitted provided that the following conditions are met:</p>
+            
+            <ul>
+                <li>Redistributions of source code must retain the above copyright
+                    notice, this list of conditions and the following disclaimer.</li>
+                <li>Redistributions in binary form must reproduce the above copyright
+                    notice, this list of conditions and the following disclaimer in the
+                    documentation and/or other materials provided with the distribution.</li>
+                <li>Neither the name of Sage Bionetworks nor the names of BridgeSDk&rsquo;s
+                    contributors may be used to endorse or promote products derived from
+                    this software without specific prior written permission.</li>
+            </ul>
+            
+            <p>THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS <q>AS IS</q> AND
+            ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+            WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+            DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> BE LIABLE FOR ANY
```

Comment:
```suggestion
            DISCLAIMED. IN NO EVENT SHALL SAGE BIONETWORKS BE LIABLE FOR ANY
```
Also the same edit should be made in README.md